### ISSUE TITLE
HorizontalPodAutoscaler Walkthrough uses `autoscaling/v1` resource?

### ISSUE THREAD

**Issue by garritfra** [kubernetes/website#43685]
In [Creating the Autoscaling declaratively](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#creating-the-autoscaler-declaratively), the `HorizontalPodAutoscaler` of `autoscaling/v1` is being used, although the wording of the document seems to mostly refer to `autoscaling/v2`.

Current code snippet:

```yaml
apiVersion: autoscaling/v1
kind: HorizontalPodAutoscaler
metadata:
  name: php-apache
spec:
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: php-apache
  minReplicas: 1
  maxReplicas: 10
  targetCPUUtilizationPercentage: 50
```

Is this intended, or should it be updated to use the `autoscaling/v2` API? In that case, it could look something like this:

```yaml
apiVersion: autoscaling/v2
kind: HorizontalPodAutoscaler
metadata:
  name: php-apache
spec:
  scaleTargetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: php-apache
  minReplicas: 1
  maxReplicas: 10
  metrics:
  - type: Resource
    resource:
      name: cpu
      target:
        type: Utilization
        averageUtilization: 50
```

**Comment by dipesh-rawat** [comment]
Page reported in issue: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/
/language en

**Comment by Shubham82** [comment]
closing this issue as it is resolved by PR #43688
/close
